bears/css/CSSCombBear.py

```diff
@@ -7,15 +7,14 @@
 
 @linter(executable='csscomb',
         output_format='corrected',
-        use_stdin=True,
```

Comment:
Read https://github.com/coala/coala/blob/master/coalib/bearlib/abstractions/Linter.py#L880 to understand how `use_stdin` works. Simply removing this line is definitely not enough. It would not work properly.

Reply:
After reading the definition out loud a couple of times, i figured i since `stdin` input is not been taken anymore, i need to return the filename myself.
I am trying to decipher further!
In the meanwhile, thanks for the help! :smile:

Reply:
If possible, please help me look for proper documentation on writing bears, i have been able to find only [a basic one](http://api.coala.io/en/latest/Developers/Writing_Native_Bears.html), i don't think that cuts out for this work, a more detailed one, if exists, should do the trick!
Thanks in advance! :smile:

Reply:
This is a linter bear so you should read http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html and http://api.coala.io/en/latest/Developers/Linter_Bears_Advanced.html. If you cannot find everything you need, read the source code ;)